app/services/idv/steps/verify_base_step.rb

```diff
@@ -240,7 +240,7 @@ def async_state_done(current_async_state)
           date_of_birth: pii_from_doc[:dob],
           address: pii_from_doc[:address1],
           ssn: pii_from_doc[:ssn],
-          failure_reason: form_response.to_h[:error_details] || form_response.errors.presence,
+          failure_reason: form_response.errors.presence,  # [:error_details] is not present here
```

Comment:
supernit: extra spacce
```suggestion
          failure_reason: form_response.errors.presence, # [:error_details] is not present here
```

Reply:
I ended up removing the comment anyways... Thanks!